lib/rubocop/cop/minitest/skip_without_reason.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks for skipped tests missing the skipping reason.
+      #
+      # @example
+      #   # bad
+      #   skip
+      #   skip('')
+      #
+      #   # bad
+      #   if condition?
+      #     skip
+      #   else
+      #     skip
+      #   end
+      #
+      #   # good
+      #   skip("Reason why the test was skipped")
+      #
+      #   # good
+      #   skip if condition?
+      #
+      class SkipWithoutReason < Base
+        MSG = 'Add a reason explaining why the test is skipped.'
+
+        RESTRICT_ON_SEND = %i[skip].freeze
+
+        def on_send(node)
+          return unless blank_argument?(node)
+
+          conditional_node = conditional_parent(node)
+          return if conditional_node && !only_skip_branches?(conditional_node)
+
+          return if node.parent&.resbody_type?
+
+          add_offense(node)
+        end
+
+        private
+
+        def blank_argument?(node)
+          message = node.arguments.first
```

Comment:
That's just my two cents. Can you tweak it?

```suggestion
          message = node.first_argument
```